pypesto/visualize/clust_color.py

```diff
@@ -0,0 +1,73 @@
+from scipy import cluster
+import matplotlib.colors as colors
+import matplotlib.cm as cm
+import numpy as np
+
+
+def get_cluster(result_fval):
```

Comment:
I found that 'result_fval' is changed to 'fvals' later in the 'parameters' function, so maybe also need to be changed here?

Reply:
I'd simply go for `x`. We don't need any interpretation here.